_data/icons/caduceus.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://bafkreigt4dptkqi57mhzcelltztb5dd26uyjreqwsl5pp3bw7ittbesuii",
```

Comment:
size does not match

identify bafkreigt4dptkqi57mhzcelltztb5dd26uyjreqwsl5pp3bw7ittbesuii
bafkreigt4dptkqi57mhzcelltztb5dd26uyjreqwsl5pp3bw7ittbesuii SVG 354x354 354x354+0+0 8-bit sRGB 11460B 0.000u 0:00.000